GOVERNANCE.md

```diff
@@ -13,27 +13,42 @@ The following code repositories are governed by the Cloud Native Buildpacks comm
 * [resources](https://github.com/buildpacks/resources)
 * [community](https://github.com/buildpacks/community)
 * [samples](https://github.com/buildpacks/samples)
+* [github-actions](https://github.com/buildpacks/github-actions)
+* [registry-api](https://github.com/buildpacks/registry-api)
 
-## [Core Team](TEAMS.md#Core-Team)
-The Core team is responsible for the direction of the project (roadmap), subteam leadership, the spec, and cross-cutting concerns. This includes:
+## [Technical Oversight Committee](TEAMS.md#Technical-Oversight-Committee)
 
-* [spec](https://github.com/buildpacks/spec)
-* [rfcs](https://github.com/buildpacks/rfcs)
+The CNB Technical Oversight Committee (TOC) is modeled on the [CNCF TOC](https://github.com/cncf/toc) as a technical governing body. It oversees all aspects of the project and has a mandate to drive consensus for:
 
-Core team members are responsible for voting on RFCs, specification updates, and other changes to the project such as changes in leadership or community roles. A simple majority is required in each case. No company will have a combined voting block of larger than 50%.
+* Defining and maintaining the technical vision for the CNB project
+* Defining the governance structure and teams of the CNB project
+* Appointing Team Leads for each CNB Team
+* Creating new teams, or removing existing teams
+* Casting binding votes on RFCs
+* Defining the annual roadmap
+* Anything else that falls through the cracks
 
-## Sub-teams
-Sub-teams are responsible for narrower sets of concerns related to specific aspects of the project. Each sub-team will include at least one core team member to help align with the broader roadmap.
+## Teams
+CNB Teams are responsible for narrower sets of concerns related to specific aspects of the project.
 
 ### Roles
 
+#### Team Leads
+
+Each team has a Team Lead. The Team Lead is a maintainer who is appointed by the TOC to be responsible for:
+
+* Casting binding votes on RFCs owned by their team
+* Moving an RFC to a status of "voting"
+* Assigning an end-date for voting on their team’s RFCs
+* Delegating another maintainer from the same Team to temporarily fulfill these responsibilities
```

Comment:
Similar to the other sections, should we add a few lines on how community members may become team leads?

(Super majority vote from TOC?)

Reply:
resolved by https://github.com/buildpacks/community/pull/165/commits/198808695694e0c6be96c63608bfc2813845d68d